fastlane-config/project_config.rb

```
@@ -0,0 +1,247 @@
+      SHARED[:firebase_service_credentials]
+    end
+
+    def self.ios_config
+      IOS.merge(IOS_SHARED)
+    end
+
+    def self.validate_config
+      required_files = [
+        SHARED[:firebase_service_credentials],
+        IOS_SHARED[:app_store_connect][:key_filepath],
+        IOS_SHARED[:code_signing][:match_git_private_key]
```

RuboCop (1.84.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)